### DISCUSSION TITLE
Nullable enum field, sending null results in error

### DISCUSSION THREAD

**Discussion by wlp2s0** [mercurius-js/mercurius#831] [Q&A]
Hello, I'm sending null in a query for a nullable Enum field, however I'm receiving from the mercurius server an error.
Specifically the mutation is:
```gql
mutation addSaleLine {
  addSaleLinesConfiguration(
    input: {
      geographicSaleLineConfigurations: [
        {
          passThroughSelection: null # nullable enum value, this errors
        }
      ]
    }
  ) {
    id
  }
}
```
Response from server:
```
{
  "errors": [
    {
      "message": "geographicSale validation failed: lineConfigurations.0.passThroughSelection: `null` is not a valid enum value for path `passThroughSelection`.",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "addSaleLinesConfiguration"
      ]
    }
  ],
  "data": null
}
```
Searching for the issue I found that there was a problem in the `graphql-js` implementation that has been fixed in 2017.
[graphql-js issue](https://github.com/graphql/graphql-js/pull/848)
I'm not sure if I did something wrong or if this is an actual bug.

**Comment by mcollina** [comment]
Thanks for reporting!

Can you provide steps to reproduce? We often need a [reproducible example](https://stackoverflow.com/help/minimal-reproducible-example), e.g. some code that allows someone else to recreate your problem by just copying and pasting it. If it involves more than a couple of different file, create a new repository on GitHub and add a link to that.
